hibernate-search-plugins/org.jboss.tools.hibernate.search.runtime.v_4_0/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,18 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: hibernate.search.runtime.v_4_0
+Bundle-SymbolicName: org.jboss.tools.hibernate.search.runtime.v_4_0;singleton:=true
+Bundle-Version: 5.2.0.qualifier
+Fragment-Host: org.jboss.tools.hibernate.runtime.v_4_0;bundle-version="5.1.2.qualifier"
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/hibernate-search-engine-4.0.0.Final.jar,

```

Comment:
Should we really ship support for this extremely old version? May I suggest you start from version 5.5.5 - which is the one in latest EAP versions - and then we add new things from there as needed?